src/CarbonAware.CLI/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
Is this set in here for debugging? If we're checking this file in, maybe we should add some comments of how you'd modify it to debug other commands